README.md

```diff
@@ -1,21 +1,21 @@
-## Sewer          
+## Sewer
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/ccf655afb3974e9698025cbb65949aa2)](https://www.codacy.com/app/komuW/sewer?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=komuW/sewer&amp;utm_campaign=Badge_Grade)
 [![CircleCI](https://circleci.com/gh/komuw/sewer.svg?style=svg)](https://circleci.com/gh/komuw/sewer)
 [![codecov](https://codecov.io/gh/komuW/sewer/branch/master/graph/badge.svg)](https://codecov.io/gh/komuW/sewer)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/komuw/sewer)
 
 
-Sewer is a Let's Encrypt(ACME) client.           
-It's name is derived from Kenyan hip hop artiste, Kitu Sewer.                        
-It allows you to obtain ssl/tls certificates from Let's Encrypt.       
+Sewer is a Let's Encrypt(ACME) client.
+It's name is derived from Kenyan hip hop artiste, Kitu Sewer.
+It allows you to obtain ssl/tls certificates from Let's Encrypt.
```

Comment:
@AlecTroemel do you mind undoing this change? The spaces at the end are important so that markdown creates a new line.